config/locales/en.yml

```diff
@@ -2590,6 +2590,10 @@ See the %{link} to find out more about %{sitename}'s features and to start using
       now_out_of_stock: is now out of stock.
       only_n_remainging: "now only has %{num} remaining."
     variant_overrides:
+      options:
+        on_demand:
+          unlimited: "On Demand"
+          limited: "Limited Stock"
```

Comment:
as per comment above, instead of unlimited/limited here I'd just use active/inactive or true/false.